index.js

```diff
@@ -0,0 +1,127 @@
+'use strict';
+
+const jwt = require('jsonwebtoken');
+const aws = require('aws-sdk');
+const _ = require('lodash');
+const async = require('async');
+
+let AwsCustomAuthoriserToolkit = {};
+
+AwsCustomAuthoriserToolkit.buildPolicyFromIam = (jwtIdToken, callback) => {
+
+    let authToken = jwtIdToken;
+    let iam = new aws.IAM();
+    let decodedJwt = jwt.decode(authToken);
+
+    console.log(decodedJwt);
```

Comment:
If the console log is unnecessary, please remove it. JWT may consists with sensitive data which will ultimately recorded in the CloudWatch logs. May be add a verbose mode for logging throughout the project.